src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1054,10 +897,11 @@ public void setManualHousing(final boolean manualHousing)
     public void removeWorkOrder(final int orderId)
```

Comment:
Should this actually remove the work order? You call `markDirty` here, but nothing done inside this method actually makes the colony dirty.

Reply:
You right it doesn't the function calling it does, have adapted this.

Reply:
changed the name, it only removes the workOrder in the view.